x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -233,7 +234,7 @@ export class ExecuteReportTask implements ReportingTask {
       const defaultOutput = null;
       docOutput.content = output.toString() || defaultOutput;
       docOutput.content_type = unknownMime;
-      docOutput.warnings = [output.details ?? output.toString()];
+      docOutput.warnings = [output.toString()];
```

Comment:
I'm not sure where we lost the ability to have multiple warnings. Just recently it looked correct to me in the code: https://github.com/elastic/kibana/pull/126358/files#diff-9050201835b339bdd86ae9b547a047ae13476e7390f951f00768bb15c689a593R229